src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj

```diff
@@ -14,7 +14,7 @@
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
 
   <PropertyGroup>
-    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(AssemblyVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
+    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(BuildVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
```

Comment:
Yes, this build number will be constructed out of the build number which will be greater than the previous versions all the time